docs/integrations/mirroring.md

```diff
@@ -175,4 +205,5 @@ Useful fields:
 * getMirrorStatistics command - a hidden command that returns mirroring statistics: total mirrored, rate limited, and last run.
 * getSyncMirrorRecords - a hidden command that returns records that hold internal mirroring metadata for each mirrored incident
 * get-remote-data - is runnable through the war room with the relevant arguments and see the results.
+* get-modified-remote-data - is runnable through the war room with the relevant arguments and see the results.
```

Comment:
you tried? running like `!get-modified-remote-data` ?

Reply:
Yes.
You can see it returned a list with one modified incident (incident id is 1):
![image](https://user-images.githubusercontent.com/50294648/99819767-8b5a4f00-2b58-11eb-830e-e875824793f5.png)

Reply:
it is runnnable only if we add this to yml as a command